torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -588,6 +598,31 @@ class TensorExprKernel {
             });
       } break;
 
+      case aten::sigmoid: {
+        return Compute("aten_sigmoid", texprDims(v), [](const Expr& a) {
+          return Expr(1.0f) / exp(a);
```

Comment:
is this actually the right expression for sigmoid?  Isn't it `1 / (1 + exp(a))`?  But if it's not the right expression, how are the tests passing?

Reply:
Thanks for catching that. That's not the right expression. Let me check the tests.

Reply:
The tests were not working.
It's because torch.rand(1024, dtype=float), creates float64 (double) tensors and were not taking our code path. Modifying the tests throws a lot of interesting mismatches between the eager mode and our implementations. I am debugging this.